app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class UserSerializer
+  include JSONAPI::Serializer
+  attributes :first_name, :last_name, :email, :password_digest
```

Comment:
I think is not necessary to show the encrypted password.
```suggestion
  attributes :first_name, :last_name, :email
```